templates/landingpage/index.html

```diff
@@ -49,19 +51,28 @@ <h1>Student/Parent</h1>
         </div>
 
         <div class="carousel-item">
-{#            TODO - add a background image url#}
             <img class="third-slide"
                  src="{% static 'landingpage/images/bg_4.jpg' %}"
                  alt="Third slide">
-{#            <div class="blur-bg"></div>#}
+			<div class="blur-bg"></div>
             <div class="container">
-                <div class="carousel-caption d-none d-md-block text-left">
+                <div class="carousel-caption justify-content-center d-flex h-50 d-md-block text-left">
                     <h1>Admission Staff</h1>
                     <p>Looking to manage millions of applications? Start now.</p>
-{#                    <p>#}
+
                     <a href="{% url 'logIn:login_user' constant_ut_adminStaff %}"><button class="btn btn-lg btn-primary">Begin Here</button></a>
-{#                    <a class="btn btn-lg btn-primary" onclick="location.href={% url 'logIn:login_user' 1 %}" role="button">Begin here</a>#}
-{#                    </p>#}
+                </div>
+            </div>
+        </div>
+         <div class="carousel-item">
+            <img class="third-slide"
+                 src="{% static 'landingpage/images/bg_4.jpg' %}"
```

Comment:
Is this code duplicated (same as the lines starting at 55)? Please make sure that the name of the "third_slide" is unique (it might hinder in the carousel's functionality, otherwise)

Reply:
> Is this code duplicated (same as the lines starting at 55)? Please make sure that the name of the "third_slide" is unique (it might hinder in the carousel's functionality, otherwise)

no worry,  this one looks good to me.